Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -82,11 +82,11 @@ def get_installation_access_token(installation_id: str, jwt_token: str):
     if response.status_code == 201:
         return response.json()['token']
     elif response.status_code == 403:
-        return_error('403 Forbidden - The credentials are incorrect')
+        return_error('403 Forbidden - The credentials are incorrect')  # noqa: RET503
     elif response.status_code == 404:
-        return_error('404 Not found - Installation wasn\'t found')
+        return_error('404 Not found - Installation wasn\'t found')  # noqa: RET503
     else:
```

Comment:
so RET503 is for implicit-return. 
So if we remove the `else` here we will fix it, no?
Because if all the above cases didn't work, we will `return_error(f'Encountered an error: {response.text}')`, right?

Reply:
After removing the `else` It leaves only the latest one

Reply:
I returned the `else` and added the `return None` to each condition.
@bziser WDYT?